tests/common/devices/multi_asic.py

```diff
@@ -429,6 +429,8 @@ def modify_syslog_rate_limit(self, feature, rl_option='disable'):
                     services.append(service_name)
 
         for docker in services:
+            if docker == "gbsyncd":
+                continue
```

Comment:
This will skip for single asic as well.
Instead of doing this, can we add a new list "ASIC_SERVICES =["gbsyncd"] and use ASIC_SERVICES list along with DEFAULT_ASIC_SERVICES in this function and https://github.com/Azure/sonic-mgmt/blob/b3556ba88a3a02d521757c7a188d02111e2cb131/tests/common/devices/sonic_asic.py#L189 and get_docker_name().

Reply:
if not, then localize this change for multi-asic and add a TODO comment to be resolved after the issue is resolved.

Reply:
can we keep this change only for multi-asic ?
if docker == "gbsyncd" and self.sonichost.is_multi_asic
And a TODO comment

Reply:
> can we keep this change only for multi-asic ? if docker == "gbsyncd" and self.sonichost.is_multi_asic And a TODO comment

sure, I agree.